common/lib/capa/capa/templates/clarification.html

```diff
@@ -0,0 +1,4 @@
+<span class="clarification">
+	<i data-tooltip="${clarification | h}" data-tooltip-show-on-click="true" class="fa fa-info-circle"></i>
```

Comment:
@cptvitamin this is an open source contributor.

@bradenmacdonald does that make sense, or do you require more detail?